src/override/vs/base/common/semver/semver.ts

```diff
@@ -1,5 +1,2 @@
-export const gte = (): boolean => false
-export const gt = (): boolean => false
-export const lt = (): boolean => false
-export const lte = (): boolean => false
-export const valid = (): boolean => false
+// @ts-ignore
```

Comment:
you can add `"@types/vscode-semver": "npm:@types/semver@=5.5.0",` in devDependencies instead

Reply:
Smart!